Geometry/HcalTowerAlgo/interface/HcalGeometry.h

```diff
@@ -100,19 +102,19 @@ class HcalGeometry : public CaloSubdetectorGeometry {
 		     Pt3D&           ref   ) ;
   
   void newCell( const GlobalPoint& f1 ,
-			const GlobalPoint& f2 ,
-			const GlobalPoint& f3 ,
-			const CCGFloat*    parm,
-			const DetId&       detId     ) override ;
+		const GlobalPoint& f2 ,
+		const GlobalPoint& f3 ,
+		const CCGFloat*    parm,
+		const DetId&       detId     ) override ;
 
-  const CaloCellGeometry* getGeometry( const DetId& id ) const override {
-      return cellGeomPtr( m_topology.detId2denseId( id ) ) ;
+  const CaloCellGeometry* getGeometryBase( const DetId& id ) const {
```

Comment:
@bsunanda - I think, it should be private member function

Reply:
I am using this in some testing code. That is why it is public. I am simply back porting here

Reply:
@iannna please look at the comment I made

Reply:
Let's keep API clean. Please, declare HcalGeometryPlan1Tester class as a friend if you plan to keep the test running.

Reply:
however, you are not back porting the test, so no need to have it public.